sdk/nodejs/tests/runtime/langhost/run.spec.ts

```diff
@@ -1418,7 +1417,7 @@ describe("rpc", () => {
                     // SupportsFeature callback
                     (call: any, callback: any) => {
                         const resp = new resproto.SupportsFeatureResponse();
-                        resp.setHassupport(false);
+                        resp.setHassupport(true);
```

Comment:
I'm not 100% sure why this was set to `false` before - do we need to support that?  If so, I can just add an opt to switch this on-and-off per test.

Reply:
I _think_ it's OK to set it to true. I don't think we have any tests that assume it's false.